lib/src/extras/launcher.rs

```diff
@@ -1,17 +1,16 @@
 use url::Url;
```

Comment:
I'm not seeing this file used anywhere. Could you consider to delete it instead?